scrapers/AlexLegend/AlexLegend.yml

```diff
@@ -0,0 +1,48 @@
+name: AlexLegend
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - alexlegend.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[contains(concat(" ", normalize-space(@class), " "), " content-title ")]/h1
+      Code:
+        selector: //link[@rel="canonical"]/@href
+        postProcess:
+          - replace:
+            - regex: .+-(\d+)\..+
+              with: $1
+      Details: //div[contains(concat(" ", normalize-space(@class), " "), " content-desc ")][2]
+      Date:
+        selector: //div[contains(concat(" ", normalize-space(@class), " "), " content-date ")]/div[contains(concat(" ", normalize-space(@class), " "), " label ")]
+        postProcess:
+          - parseDate: 02.01.2006
+      Image:
+        selector: //script[contains(text(),"myFluidPlayer")]
+        postProcess:
+          - replace:
+            - regex: .+posterImage\s?:\s?'(.*?)'.+
+              with: $1
+      Studio:
+        Name:
+          fixed: AlexLegend
+      Tags:
+        Name:
+          selector: //div[contains(concat(" ", normalize-space(@class), " "), " content-tags ")]
+          split: ", "
+          postProcess:
+            - replace:
+              - regex: 'Tags:'
+      Performers:
+        Name:
+          selector: //div[contains(concat(" ", normalize-space(@class), " "), " main__models ")]

```

Comment:
Could've just grabbed `@title` of the element instead of stripping nbsp and `,`

Reply:
fixed